BranchMonsterFactory/Images.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -1,118 +0,0 @@
-{
```

Comment:
i cloned a fresh copy, switched to this branch, and can't get it to run because this is missing